.github/actions/sls-deployment/action.yml

```diff
@@ -0,0 +1,88 @@
+name: sls deploy action
+description: Reusable sls deploy
+
+inputs:
+  environment-name:
+    description: 'The environment name'
+    required: true
+  app:
+    description: 'The application - crn-server or gp2-server'
+    required: true
+  AWS_ACCESS_KEY_ID:
```

Comment:
In the other actions we're tried to make the inputs kebab cased. Can we do the same here?